gradle-jdks-distributions/build.gradle

```diff
@@ -7,4 +7,6 @@ dependencies {
 
     testImplementation 'org.junit.jupiter:junit-jupiter'
     testImplementation 'org.assertj:assertj-core'
+
+    testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
```

Comment:
I don't think these manually need to be added after https://github.com/palantir/gradle-baseline/pull/3053